bolt/database.types.bolt

```diff
@@ -11,7 +11,8 @@ type Event {
   location: String | Null,
   mapsUrl: String,
   createdBy: String,
-  createdAt: Number
+  createdAt: Number,
+  firebaseId: String
```

Comment:
This `firebaseId` value is already present on this model - it's `createdBy`. The field should probably be labeled `createdByUid`.